added asteroids

Mokimbo opened the pull request:
put some static asteroids on the map to run into for now. can talk about what happens to them when you hit them next meeting.
Also added the image for the asteroids so we can code in some variation later.

Mokimbo committed:
added asteroids

alarner reviewed (changes requested):
Looks great! Only comment is we should probably put the meteor folder in the new art directory

j-rodriguez99 commented:
If the player flies a bit the background acts strangely. Also, I noticed you used a rigid body but set the mode property to static. I'm not to familiar with the mode setting but was curious about that choice.

j-rodriguez99 commented on World.tscn:
```diff
@@ -32,6 +160,7 @@ motion_scale = Vector2( 0.2, 0.2 )
 motion_mirroring = Vector2( 4096, 4096 )
 
 [node name="Sprite" type="Sprite" parent="ParallaxBackground/ParallaxLayer1"]
+scale = Vector2( 1.5, 1.5 )
```
I'm wondering if changing the scale here will have an impact on the mirroring of the background. 
```suggestion
scale = Vector2( 1, 1 )
```

Mokimbo replied:
if i left it as the default rigid body you could push it around when you hit it. I think its a cool idea for later on when we get them to move on their own that you could knock them out of their velocity, or just have them weigh a lot so you slow down when you hit them. Setting it to static just makes them act like a static body like the planet is.

Mokimbo committed:
changed parallax1 scale, started to sort images

Mokimbo committed:
organized art into folders

Mokimbo committed:
moved backrounds to art and fixed game sprites

Mokimbo commented:
I moved all the art into their respective folders so it should be easier to find things your looking for. Also fixed the background so it looks better like josh suggested, it does indeed look much better.

alarner reviewed (approved):
LGTM
